action.yml

```diff
@@ -156,10 +156,13 @@ runs:
       echo ::group::Move "'${{ steps.collection-metadata.outputs.fqcn }}'"
       collection to ${{ steps.collection-metadata.outputs.checkout-path }}...
     shell: bash
-  - name: Install ansible-core (${{ inputs.ansible-core-version }})
+  - name: Move collection to test directory
     run: >-
       set -x
       ;
+      rm -rf 
+      "${{ steps.collection-metadata.outputs.checkout-path }}"
```

Comment:
This should:
1) Clean up the whole directory structure we manage to prevent any further side effects.
2) Do (1) as the very first step in this action, before the checkout.

Reply:
Moreover, I'd probably clean up both dirs, I think. Maybe the whole collections dir too.

Reply:
That makes sense, I've implemented the change per your suggestion.